docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  db:
+    image: mdillon/postgis
+  web:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/Dockerfile
+    command: python manage.py runserver 0.0.0.0:8000
+    ports:
+      - "8000:8000"
+    depends_on:
```

Comment:
Replace this with a `links` section. It creates a dependency tree and gives the container direct access to the other container.
```
links:
  - db
```